scripts-bootstrap.js

```diff
@@ -0,0 +1,24 @@
+require('shelljs/global')
+
+mkdir('-p', 'typings')
+
+touch('typings/main.d.ts')
+
+echo('SKIP_PREPUBLISH=true')
+env['SKIP_PREPUBLISH'] = true
+
+exec('npm install')
+exec('rimraf dist')
+
+if (!which('tsc')) {
```

Comment:
Yeah, I didn't see it mentioned either. Do you have `rimraf` installed globally?